RF24.h

```diff
@@ -598,6 +598,46 @@ class RF24 {
      * a `1` means the feature is enabled.
      */
     void printPrettyDetails(void);
+    
+    /**
+     * Put a giant block of debugging information in a char array. This function
+     * differs from printDetails() because it makes the information more
+     * understandable without having to look up the datasheet or convert
+     * hexadecimal to binary. Only use this function if your application can
+     * spare extra bytes of memory.
+     *
+     * @warning use a buffer of sufficient size for debugging_information, start 
+     * with a sizeof >= 1024
+     ^ @param debugging_information The c-string buffer that the debugging 
+     *  information is stored to.
+     * 
+     * @note http://www.cplusplus.com/reference/cstdio/sprintf/
```

Comment:
Please change `@note` to `@remark`. This is to
1. avoid doxygen drawing too much attention for otherwise basic C feature (like using `sprintf()`)
2. I don't want to pollute the doc with unnecessary admonished text.

This is not a mistake on your behalf. Its just me being very sensitive to the docs I've put so much work into.

Reply:
I understand and made the requested changes.